src/index.js

```diff
@@ -1,8 +1,3 @@
 import SmartBanner from './smartbanner.js';
 
-let smartbanner;
-
-window.addEventListener('load', function() {
-  smartbanner = new SmartBanner();
-  smartbanner.publish();
-});
+window.SmartBanner = SmartBanner;
```

Comment:
To avoid the breaking change and keep the component elegant for non-programmatic use I'd like to suggest:

1. using `window.smartbanner`
2. introduce additional meta tag which controls API usage (manual construction),  e.g.:

    ```html
    <meta name="smartbanner:api" content="true">
    ```

    which defaults to `false`.